docs/tutorials/multinode/README.md

```diff
@@ -0,0 +1,332 @@
+
+##  Step 3: Run terraform
+
+After the previous steps are completed, you can initialize `terraform` to begin your cluster creation.
+The first step is to initialize the `terraform` state.
+``` bash
+terraform init
+```
+A successful result will contain the text:
+```
+Terraform has been successfully initialized!
+```
+### Run the `make` command
+For convenience, some terraform commands are prepared in a `Makefile`. This means
+you can now create your cluster, with a simple `make` command.
+```bash
+make apply
+```
+A successful run will show:
+```
+Apply complete! Resources: 4 added, 0 changed, 0 destroyed.
+```
+## Step4: Connect to the _submit_ node for HTCondor
+Although there are ways to run `HTCondor` commands from your local machine, 
+the normal path  is to login to the _submit_ node. From there you can run 
+commands to submit and monitor jobs on HTCondor.
+
+### List VMs that were created by
+
+You can list the VMs created. One of them will be the submit node. It will be the VM with
+"submit" in the name.
+
+```
+gcloud compute instances list
+```
+You will see two instances listed:
+* c-manager: the core controller node
+* c-submit: the submit node where you will run interactive work.
+
+Identify then log in to the `submit` node. If you used the standard install, the
+cluster name is "c". In that case you would connect using the `gcloud ssh`
+command.
+```bash
+gcloud compute ssh c-submit
+```
+Now you are logged in to your HTCondor cluster. You will see a command prompt
+something like:
+```bash
+[mylogin@c-submit ~]$
+```
+
+### Checking the status 
+You can verify if the HTCondor install is completed:
+```
+condor_q
+```
+You will see output:
+```
+-- Schedd: c-submit.c.quantum-htcondor-14.internal : <10.150.0.2:9618?... @ 08/18/21 18:37:50
+OWNER BATCH_NAME      SUBMITTED   DONE   RUN    IDLE   HOLD  TOTAL JOB_IDS
+
+Total for query: 0 jobs; 0 completed, 0 removed, 0 idle, 0 running, 0 held, 0 suspended
+Total for drj: 0 jobs; 0 completed, 0 removed, 0 idle, 0 running, 0 held, 0 suspended
+Total for all users: 0 jobs; 0 completed, 0 removed, 0 idle, 0 running, 0 held, 0 suspended
+```
+If you get `command not found`, you will need to wait a few minutes for the HTCondor install to complete.
+
+## Step 5: Get the sample code and run it
+The HTCondor cluster is now ready for your jobs to be run. If you are familiar with HTCondor and you want to run your own jobs, you may do so. 
+
+If you don't have jobs to run, you can get sample jobs from this Github repo. You will clone the repo to 
+the submit node and run a job.
+
+###  Clone the repo on your cluster
+
+on the `submit` node you you can install the repo to get access to previously created submission files:
+```
+git clone https://github.com/jrossthomson/qsim.git
+```
+Then cd to the tutorial directory.
+```
+cd qsim/docs/tutorials/multinode
+```
+### Submit a job
+Now it is possible to submit a job:
+```
+condor_submit circuit_q24.sub
```

Comment:
If possible, we should avoid using examples based on the qsim text input format. This format is deprecated, and switching between it and Cirq-based input will only confuse the user.